<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STranslate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/translate.png</normaloff>:/image/translate.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="mianLayout">
      <item row="4" column="1" colspan="3">
       <widget class="QTextEdit" name="translateTextEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="translateLabel">
        <property name="text">
         <string>Translate text:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fromComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fromLabel">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="toComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="appIDLabel">
        <property name="text">
         <string>AppID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="srcLabel">
        <property name="text">
         <string>Src text:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="keyLabel">
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="appIDLineEdit"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="keyLineEdit"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QTextEdit" name="srcTextEdit">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;苹果&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
  </widget>
  <widget class="QToolBar" name="translateToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="translateAction"/>
   <addaction name="clearAction"/>
  </widget>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="translateAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/translate.png</normaloff>:/image/translate.png</iconset>
   </property>
   <property name="text">
    <string>translate</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/clear.png</normaloff>:/image/clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
